src/swap/core/getSwapErrorCode.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  GENERAL_SWAP_ERROR_CODE,
+  GENERAL_SWAP_BALANCE_ERROR_CODE,
+  GENERAL_SWAP_QUOTE_ERROR_CODE,
+  LOW_LIQUIDITY_ERROR_CODE,
+  UNCAUGHT_SWAP_ERROR_CODE,
+  UNCAUGHT_SWAP_QUOTE_ERROR_CODE,
+} from '../constants';
+
+export function getSwapErrorCode(
```

Comment:
Next PR, we should find the right place to document those error in Swap.

Maybe as we have the `types` page, we start having the `errors` page for each Module. Which will help inform the type of error components can have for each module.